Instant-Gaming/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+  "author": {
+    "name": "Dekirai",
+    "id": "601560798409654292"
+  },
+  "url": "www.instant-gaming.com",
+  "description": {
+    "en": "Instant-Gaming.com - All your favourites games for Steam, Origin, Battle.net, Uplay and Indie games up to 70% off! Digital games, Instant delivery 24/7!",
+    "de": "Instant-Gaming.com - Deine Lieblingsspiele für Steam, Origin, Battle.net, Uplay und Indie Games mit bis zu 70% Rabatt! Digitaler Download, Sofortige Lieferung, 24/7!"
+  },
+  "service": "Instant Gaming",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/ODMBHzO.png",
```

Comment:
Please get a higher quality version of the logo or use [waifu2x](http://waifu2x.udp.jp/index.pt.html)